CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # CHANGELOG
 
 ## 0.11.1dev
-
+- [Feature] Improves composed ROC plots so that different curves are more easily identfiable
```

Comment:
```suggestion
- [Feature] Improves composed ROC plots so that different curves are more easily identifiable
```

Reply:
Done! Thanks, sorry didn't even catch that